nilearn/glm/tests/test_first_level.py

```diff
@@ -2228,6 +2213,28 @@ def test_flm_compute_contrast_with_surface_data(_make_surface_glm_data):
     assert isinstance(result, SurfaceImage)
 
 
+def test_flm_get_voxelwise_model_attribute_with_surface_data(
+    surface_glm_data,
+):
+    """Smoke test 'voxel wise' attribute with surface data.
+
+    TODO: rename the private function _get_voxelwise_model_attribute
+    to work for both voxel and vertex
```

Comment:
TODO: private function should be renamed

Reply:
you leave it for a future PR ?

Reply:
yes: this PR is big enough